client/ayon_core/hosts/nuke/plugins/create/create_write_render.py

```diff
@@ -40,11 +40,15 @@ def get_pre_create_attr_defs(self):
         return attr_defs
 
     def create_instance_node(self, product_name, instance_data):
+        settings = self.project_settings["nuke"]["create"]["CreateWriteRender"]
+        settings = settings["instance_attributes"]
+
         # add fpath_template
         write_data = {
             "creator": self.__class__.__name__,
             "productName": product_name,
-            "fpath_template": self.temp_rendering_path_template
+            "fpath_template": self.temp_rendering_path_template,
+            "headless_farm_submission": "headless_farm_submission" in settings
```

Comment:
Is this being present in settings really what makes it a headless farm submission? Shouldn't it check a value or whatever? Or how does this work?

Reply:
The `settings` variable here is an enum; `ayon+settings://nuke/create/CreateWriteRender/instance_attributes?project=demo_Commercial`

![Screenshot 2024-04-12 155650](https://github.com/ynput/ayon-core/assets/1860085/27dca3aa-e65e-4034-a7a5-e34fda0f5bdd)

Reply:
I see now - ok, then maybe `settings` as a variable name isn't very explicit and `instance_attributes` as variable name may be more accurate?

Reply:
> I see now - ok, then maybe settings as a variable name isn't very explicit and instance_attributes as variable name may be more accurate?

100% agree
```
        instance_attributes = (
            self.project_settings
            ["nuke"]
            ["create"]
            ["CreateWriteRender"]
            ["instance_attributes"]
        )
```

Reply:
Done